daml-script/daml/Daml/Script/Free.daml

```diff
@@ -0,0 +1,34 @@
+-- Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+module Daml.Script.Free
```

Comment:
```suggestion
-- | HIDE
module Daml.Script.Free
```

Given that we deliberately hide this from the public API, we should also hide it from the docs. We could also follow the pattern that we use in the trigger library and add an `Internal` in the module name to make it very explicit that this is not part of the supported API.